deny.toml

```diff
@@ -48,25 +48,21 @@ skip-tree = [
 
     # wait for zcashd and zcash_script to upgrade
     # https://github.com/ZcashFoundation/zcash_script/pulls
-    { name = "zcash_primitives", version = "=0.9.1" },
+    { name = "metrics", version = "=0.20.1" },
     { name = "sha2", version = "=0.9.9" },
-    { name = "reddsa", version = "=0.3.0" },
 
     # wait for ed25519-zebra, indexmap, metrics-util, and metrics to upgrade
     # ed25519-zebra/hashbrown: https://github.com/ZcashFoundation/ed25519-zebra/pull/65
     { name = "ahash", version = "=0.7.6" },
 
     # ECC crates
 
-    # Wait until `orchard` updates `aes`, which depends on `cipher`
-    { name = "cipher", version = "=0.3.0" },
-
     # wait for zcash_primitives to remove duplicated dependencies
     { name = "block-buffer", version = "=0.9.0" },
```

Comment:
This dependency is via `sha2 0.9.9` so it should go away with https://github.com/zcash/zcash/issues/6648 .

Reply:
@teor2345 should we wait for this one before merging this PR?

Reply:
actually I guess not since this will be in zcash 5.6.0 and I don't think we want to wait that long

Reply:
Duplicate dependencies usually aren't a blocker, unless they're consensus-critical and we're actually calling both.